gtsam_unstable/examples/FixedLagSmootherExample.cpp

```diff
@@ -145,5 +145,18 @@ int main(int argc, char** argv) {
     cout << setprecision(5) << "    Key: " << key_timestamp.first << "  Time: " << key_timestamp.second << endl;
   }
 
+  // get the linearization point
```

Comment:
I understand this is the code that diagnoses the problem. However, this file is an example. If we leave the code below, comment the block to say what this is an example of :-)